### PR TITLE
Banner added

### PR DESCRIPTION
**Pull request by aditisneh**
## What is the change?
Banner is added in the readme

## Related issue?
close: #745

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [yes] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [yes] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [yes] Did you lint your code before making the Pull Request?
- [] Have you made corresponding changes to the documentation?
- [] Your submission doesn't break any existing feature.
- [] Have you tested the code before submission?

### CODE REVIEW

**Comment by aditisneh** [comment]
do add proper level tag

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div align="center">
-  <h1>WordNook</h1>
```
Comment (Pranav016) [inline README.md:2]: Don't remove this for now.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
I think it could be better
image


The one we had previously looks really good, kind of the same format would do the job.

image


* For this issue I would appreciate if you first get approval in the chat itself if the banner is suitable and then make a PR. That would save your time.
* When it does get approved, remove the previous banner before adding a new banner.
* Add it only to the assets folder, in this PR the banner was added twice.

**Comment by aditisneh** [comment]
Okay, I understood that. I thought of something else. As you want me to design new banner with previous format, I think first we should work on the logo, so that same logo goes in the banner, right? Thanks for such a good explanation :)

**Comment by Pranav016** [comment]
Closing the PR for now, comment on this to reopen PR if required

**Pull request closed without merging**